index.js

```diff
@@ -4,8 +4,8 @@ const handlePullRequestChange = require('./lib/handle-pull-request-change')
 
 function probotPlugin (robot) {
   robot.on([
-	'pull_request.opened',
-	'pull_request.edited',
-	'pull_request.synchronize'
+  'pull_request.opened',
+  'pull_request.edited',
+  'pull_request.synchronize'
```

Comment:
you will need 4 spaces instead of 2. I’m sorry for the trouble, the linter is very strict about the indentation. I can also fix it up for you if you like, just let me know. The code change itself is correct